### PR TITLE
Add a flag to disable the rsyslog workers.

### PR DESCRIPTION
**Pull request by howbazaar**
Have a feature flag that disables the rsyslog workers.

(Review request: http://reviews.vapour.ws/r/1993/)

### CODE REVIEW

File: feature/flags.go [hunk 1]
```diff
@@ -53,6 +53,10 @@ func IsDbLogEnabled() bool {
 	return featureflag.Enabled(dbLog) || featureflag.Enabled(JES)
 }
 
+// DisableRsyslogLogSending will stop the writing of the rsyslog
+// accumulation and forwarding configuration files.
+const DisableRsyslogLogSending = "disable-rsyslog"
```
Comment (mjs) [inline feature/flags.go]: DisableRsyslog would have been fine but whatever...

**Comment by mjs** [comment]
LGTM

**Comment by howbazaar** [comment]
$$merge$$

**Comment by howbazaar** [comment]
$$bad-record-mac$$

**Pull request merged**